internal/controller/datadogagent/controller_v2_test.go

```diff
@@ -1157,7 +1158,13 @@ func Test_Control_Plane_Monitoring(t *testing.T) {
 			want:    reconcile.Result{RequeueAfter: defaultRequeueDuration},
 			wantErr: false,
 			wantFunc: func(t *testing.T, c client.Client) error {
-				return verifyDCADeployment(t, c, resourcesName, resourcesNamespace, dcaName, "openshift")
+				if err := verifyDCADeployment(t, c, resourcesName, resourcesNamespace, dcaName, "openshift"); err != nil {
+					return err
+				}
+				expectedDaemonsets := []string{
+					string("foo-agent"),
+				}
+				return verifyDaemonsetNames(t, c, resourcesNamespace, dsName, expectedDaemonsets)
```

Comment:
I don't think `dsName` is used or needed here since we pass `expectedDaemonsets`.

Reply:
removed in [70f40a2](https://github.com/DataDog/datadog-operator/pull/2114/commits/70f40a294b0d8cf3cc9584a5d0af501c2003fc76)